src/webnn_native/dml/GraphDML.cpp

```diff
@@ -1814,13 +1814,12 @@ namespace webnn_native::dml {
         return {};
     }
 
-    WNNComputeGraphStatus Graph::ComputeImpl(NamedInputsBase* inputs, NamedOutputsBase* outputs) {
+    MaybeError Graph::ComputeImpl(NamedInputsBase* inputs, NamedOutputsBase* outputs) {
         auto namedInputs = inputs->GetRecords();
         for (auto& [name, inputBinding] : mInputBindingMap) {
             // All the inputs must be set.
             if (namedInputs.find(name) == namedInputs.end()) {
-                dawn::ErrorLog() << "The input must be set.";
-                return WNNComputeGraphStatus_Error;
+                return DAWN_INTERNAL_ERROR("All inputs must be set.");
```

Comment:
Use DAWN_INVALID_IF macro instead of L.1821 and 1822

Reply:
Good suggestion!